README.md

```diff
@@ -17,6 +17,10 @@ should be able to preview your changes at http://localhost:4567
 
     bundle exec middleman server
 
+Dockerfile is included for processing PHP content. To run the PHP server execute the following from the root repo directory:
```

Comment:
Having trouble with this, hitting 403s with apache running as described above. Overall, looks like an over-complicated thing to me, just for a local testing (as it needs stuff [like this](https://github.com/openshift/openshift.org/pull/68/files#diff-f828450012b9c91ffbf63af17fa0c66cR11)). Why not using something like [middleman-php](https://github.com/lord/middleman-php) for development?